ebill_paynet/components/api.py

```diff
@@ -0,0 +1,51 @@
+# import base64
+import os
+
+from lxml import html
+from requests import Session
+from zeep import Client, Settings
+from zeep.transports import Transport
+
+WSDL_DOC = os.path.join(os.path.dirname(__file__), 'wsdl', 'DWSPayNet.wsdl')
+SSL_PROD_CERTIFICATE = os.path.join(
+    os.path.dirname(__file__), 'certificats', 'prod_services_chain.pem'
+)
+SSL_TEST_CERTIFICATE = os.path.join(
+    os.path.dirname(__file__), 'certificats', 'test_services_chain.pem'
+)
+
+
+class PayNetDWS(object):
```

Comment:
No need to inherit from `object` with Python 3.

Reply:
Thanks fixed